lib/src/arithmetic/values/floating_point_values/floating_point_value.dart

```diff
@@ -458,6 +521,8 @@ class FloatingPointValue implements Comparable<FloatingPointValue> {
     return true;
   }
 
+  // TODO(desmonddak): issue#203 subnormal is inaccurate.
```

Comment:
nit: good idea to include a link to the filed bug in the TODO

Reply:
looks good, usually comes at the end but no big deal

https://dart.dev/tools/linter-rules/flutter_style_todos